.github/workflows/auto.yml

```diff
@@ -0,0 +1,35 @@
+name: Automatic Build
+
+on:
+  push:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+      - name: Clone publicgoods-website
+        run: git clone https://github.com/unicef/publicgoods-website.git ../publicgoods-website
+      - name: Clone publicgoods-candidates
+        run: git clone https://github.com/unicef/publicgoods-candidates.git ../publicgoods-candidates
+      - name: Run static.bash
+        run: bash scripts/static.bash
+      - name: Generating DPG's
+        run: cd packages/automation && npm install && node generate_dpgs.js
+      - name: Generating Nominees
+        env:
+            CLIENTID: ${{ secrets.CLIENTID}}
+            CLIENTSECRET: ${{ secrets.CLIENTSECRET}}
+        run: cd packages/automation && node generate_nominees.js
```

Comment:
I have mixed feelings about this:
on the one hand, I understand that to run a proper CI, we should run this every time to test that `generate_nominees.js` runs as expected, but on the other hand, this task takes a long time to complete (several minutes) which is a drag when you are doing development. Curious to hear your thoughts @nathanfletcher

Reply:
Yes, it does take long to run, and it is a drag. We could do a couple of things:
- Conclude that it works for now and not include it in the build
- Have a separate scheduled periodic build for it so that it is not in this main one, since it is a more matured codebase.

Reply:
Agreed. Let's leave it out for now. If we feel it needs periodic testing we can add a scheduled build later, but I would mark that as low priority.